cloudbuild.yaml

```diff
@@ -14,6 +14,21 @@ steps:
     - --output=type=registry
     - --platform=linux/amd64,linux/arm64
     - .
+  # Build cloudbuild artifacts (for attestation)
+  - name: 'docker.io/library/golang:1.18.4-bullseye'
+    id: cloudbuild-artifacts
+    entrypoint: make
+    env:
+    # _GIT_TAG is not a valid semver, we use CI=1 instead
+    # - VERSION=$_GIT_TAG
+    - CI=$_CI
+    - PULL_BASE_REF=$_PULL_BASE_REF
+    - DOCKER_REGISTRY=$_DOCKER_REGISTRY
+    - DOCKER_IMAGE_PREFIX=$_DOCKER_IMAGE_PREFIX
+    - GCS_LOCATION=$_GCS_LOCATION
+    - LATEST_FILE=markers/${_PULL_BASE_REF}/latest-ci.txt
+    args:
+    - cloudbuild-artifacts
```

Comment:
We need to add this target in our Makefile, either in this PR or a separate PR that is merged first.

Reply:
added in https://github.com/kubernetes/cloud-provider-aws/pull/439